src/components/DisplayUnitTag.tsx

```diff
@@ -40,7 +42,7 @@ export const DisplayUnitTag = ({ entryPoint, assetId, className }: ElementProps)
     <Tag onClick={onToggle} tw="cursor-pointer" className={className}>
       {
         {
-          [DisplayUnit.Asset]: assetId,
+          [DisplayUnit.Asset]: getDisplayableAssetFromBaseAsset(assetId),
```

Comment:
I wonder if this tag should have a max size and wrap contents in a textTruncate

Reply:
I guess we render these so often it would be a pain. Better to do any necessary shortening in the getDisplayableAssetFromBaseAsset function probably.

Reply:
In general I think we should promise the asset name isn't longer than ~10 characters

Reply:
I noticed that even with ~8 chars it messes up the Orderbook headers (depending on tickSize) of market

Reply:
what about assets with just genuinely long names? may be worth (in the future) to add a tooltip for this? 🤔